docs/tbdev_getting_started.ipynb

```diff
@@ -12,7 +12,7 @@
     },
```

Comment:
<p>Extra newline before "You can list...".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
This will only appear in the pre-markdown format as a line break.  It does not appear as rendered for the user.

![image](https://user-images.githubusercontent.com/547150/89841265-812fba00-db40-11ea-833d-607d66fd0371.png)